.github/workflows/selfdrive_tests.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       run: |
         cd $STRIPPED_DIR
         ${{ env.RUN }} "release/check-dirty.sh && \
-                        MAX_EXAMPLES=5 $PYTEST -m 'not slow' selfdrive/car"
+                        MAX_EXAMPLES=5 $PYTEST -m 'not slow' openpilot/selfdrive/car"
```

Comment:
hmm, do we need to do this for all invocations now? if so, might be better to switch to a list of selfdrive, system, tools, and common instead of openpilot/

Reply:
we might be able to use `--rootdir` so that we can set the root to be `openpilot`. Next commit will try this.